src/system/AdminModule/Controller/AdminInterfaceController.php

```diff
@@ -0,0 +1,785 @@
+{	
+	/**
+	 * @Route("/header")
+	 *
+	 * @Template()
+	 * @Theme("admin")
+	 *
+	 * Open the admin container
+	 *
+	 * @return Response symfony response object
+	 */
+	public function headerAction()
+	{
+		$masterRequest = $this->get('request_stack')->getMasterRequest();
+		$caller = array();
+		$caller['_zkModule'] = $masterRequest->attributes->get('_zkModule');
+		$caller['_zkType'] = $masterRequest->attributes->get('_zkType');
+		$caller['_zkFunc'] = $masterRequest->attributes->get('_zkFunc');	 
+		return ['caller' => $caller];
+	}
+	
+	/**
+	 * @Route("/footer")
+	 *
+	 * @Template()
+	 * @Theme("admin")
+	 *
+	 * Close the admin container
+	 *
+	 * @return Response symfony response object
+	 */
+	public function footerAction()
+	{
+		return ['symfonyversion' => \Symfony\Component\HttpKernel\Kernel::VERSION];
+	}
+	
+	/**
+	 * @Route("/help")
+	 *
+	 * @Template()
+	 * @Theme("admin")
+	 *
+	 * display the module help page
+	 *
+	 * @return Response symfony response object
+	 *
+	 * @throws AccessDeniedException Thrown if the user doesn't have admin permission to the module
+	 */
+	public function helpAction()
+	{
+		if (!$this->hasPermission('ZikulaAdminModule::', '::', ACCESS_ADMIN)) {
+			throw new AccessDeniedException();
+		}
+	
+		return ['symfonyversion' => \Symfony\Component\HttpKernel\Kernel::VERSION];
+	}	
+	
+	/**
+	 * @Route("/breadcrumbs")
+	 * @Method("GET")
+	 *
+	 * @Template()
+	 * @Theme("admin")
+	 *
+	 * Admin breadcrumbs
+	 *
+	 * @return Response symfony response object
+	 */
+	public function breadcrumbsAction()
+	{		 
+		if (!$this->hasPermission('ZikulaAdminModule::', '::', ACCESS_ADMIN)) {
+			throw new AccessDeniedException();
+		}
+		
+		$masterRequest = $this->get('request_stack')->getMasterRequest();
+		$requested_cid = $masterRequest->attributes->get('acid');
+		$caller = [];
+		$caller['_zkModule'] = $masterRequest->attributes->get('_zkModule');
+		$caller['_zkType'] = $masterRequest->attributes->get('_zkType');
+		$caller['_zkFunc'] = $masterRequest->attributes->get('_zkFunc');
+		$caller['info'] = \ModUtil::getInfoFromName($caller['_zkModule']);
+				
+		if($caller['_zkModule'] == 'ZikulaAdminModule'){
+			$cid = empty($requested_cid) ? $this->getVar('startcategory') : $requested_cid;			
+		}else{			
+			$cid = \ModUtil::apiFunc('ZikulaAdminModule', 'admin', 'getmodcategory', array('mid' => $caller['info']['id']));
+		}			
+		$caller['category'] = \ModUtil::apiFunc('ZikulaAdminModule', 'admin', 'getCategory', array('cid' => $cid));
+		
+		return ['caller' => $caller];
+	}
+	
+	/**
+	 * @Route("/developernotices")
+	 *
+	 * @Template()
+	 * @Theme("admin")
+	 *
+	 * Open the admin container
+	 *
+	 * @return Response symfony response object
+	 */
+	public function developernoticesAction()
+	{
+		if (!$this->hasPermission('ZikulaAdminModule::', '::', ACCESS_ADMIN)) {
+			throw new AccessDeniedException();
+		}		 
+
+		$modvars = \ModUtil::getVar('ZikulaThemeModule');
```

Comment:
I think not. I don't know if we want to make it *easy* for a module to change the vars of a different module. opinions @cmfcmf @Guite ? I wasn't sure about suggesting the first :smile: